hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenAirdropHandler.java

```diff
@@ -104,7 +104,10 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
         var convertedOp = CryptoTransferTransactionBody.newBuilder()
                 .tokenTransfers(op.tokenTransfers())
                 .build();
-        preHandle(context, convertedOp);
+        // Any receiver that has `receiverSigRequired` will be ignored during airdrops.
```

Comment:
I thought this change was merged

Reply:
The PR was targeting 13658-add-TokenAirdropHandler-additional-tests, so its not yet merged in develop